plugins/epos/config.ini

```diff
@@ -142,7 +142,20 @@ terms_data_model = []
 #metadata standard
 metadata_standard = ['DCAT']
 
-
+#terms that use vocabularies and vocabualries used
+vocabularies=['ROR','PIC','imtypes','TRL','spatial','temporal','Rolecode','spdx','ORCID']
+
+vocabularies_identifiers =['https://ror.org/','https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/how-to-participate/participant-register','https://www.iana.org/assignments/media-types/media-types.xhtml','TRL','spatial','https://www.iso.org/iso-8601-date-and-time-format.html','Rolecode','https://spdx.org/licenses/','https://orcid.org/']
```

Comment:
shouldn't be better using a mapping here, like a dict in Python or JSON? is "spatial" a vocabulary?

Reply:
agree, at least for the time being